README.md

```diff
@@ -16,12 +14,50 @@
 ## Writing Regular Expressions
 
-### Part 1: Do Some Stuff
+In Ruby, regular expressions are always written between forward slashes:
+`/your regex/`. In Rubular, you can see that these slashes have already been written for you.
+
+### Simple Text Matching
+
+Let's start with the simplest text matching. Add the following regex in rubular:
+```
+/twenty/
+```
+
+![twenty regex](https://curriculum-content.s3.amazonaws.com/web-development/Regex/twenty.png)
+
+Notice that the pattern matches the two instances of "twenty" in the passage. Writing a series of letters or numbers in your regular expression will result in a search for exact matches of this pattern anywhere in the string. 
+
+### Metacharacters
+The real beauty of Regular Expressions is revealed in its use of metacharacters. Metacharacters allow you to use a pre-defined shorthand to match specific characters. For example, `\d` will match any digit in your text, and `\w` will match any word character (non-digit, non-punctuation). The 'Regex Quick Reference' at the bottom of Rubular shows metacharacters and patterns that you can use. Play around with these a little. Use `\W` (notice uppercasing) to match the non-word characters in your text.
+
+### Only specific characters
+
+If I want to match all instances of vowels in a string, the regex `/aeiou/` won't work (feel free to try it), as it will only match the entire string "aeiou" - which clearly isn't in our text. Instead let's use square brackets: `/[aeiou]/` - this is looking for only **one single** character in our text which matches any of the characters inside the square brackets. If you add this regext to our rubular, you'll see every vowel highlighted in your match result.
+
+### Ranges
+
+Based on what we've just learned, we can write a regular expression looking for single characters in the first 10 letters of the alphabet like so:`/[abcdefghij]/`
+We can actually shorten this in ruby using a Regex range:`/[a-j]/`
+
+`[0123456789]` becomes `[0-9]`
+
+### Example: Double Vowels
+
+There are many other metacharacters and ways of building patterns in regex, many of which you can refer in the Rubular quick reference guide. However, the best way to actually learn to use Regular Expressions is to practice building your own patterns. Let's look for instances in our text of two consecutive vowels. The longest way to do this is to hand code the different combinations of two vowels:`/aa|oo|ee|ii|uu|ae|ea/`. It's pretty tedious to hand code each of these combinations (I didn't finish). An improvement is to use two sets of square brackets with vowels, each one representing a single character: `/[aeiou][aeiou]/`. Our most efficient, however, is to use repetitions: `/[aeiou]{2}/` The curly braces surrounding mean that the pattern or character directly preceding it must repeat that number of times. As such, we're looking for a repeat of a vowel two times. As you can see, there are many ways to write a Regular Expression that does the same thing. 
```

Comment:
`/[aeiou]{2}/` matches aa as well as ai. To match a previous vowel you'd have to do something like (WARNING: CRAZY) `/([aeiou])\1/` meaning, match the previous value that was captured.